openslides_backend/action/actions/agenda_item/agenda_creation.py

```diff
@@ -56,18 +56,23 @@ def check_dependant_action_execution_agenda_item(
         )
         agenda_item_creation = meeting.get("agenda_item_creation")
         agenda_create = instance.pop("agenda_create", None)
+        result_value: Optional[bool] = None
         if agenda_item_creation == "always":
             return True
```

Comment:
I would favor a uniform approach and also use `result_value` here

Reply:
done, see above